website/docs/tutorials/08-interacting-with-starknet/04-deploy.md

```diff
@@ -1,29 +1,26 @@
 # Deploying new contracts
 
 ## Overview
-Protostar supports deploying smart contracts to a given network with the `protostar deploy` command. 
-It has a similar interface to the `starknet deploy` command. 
+Protostar supports deploying smart contracts to a given network with the `protostar deploy` command.
+
+It works by invoking a [Universal Deployer Contract](https://community.starknet.io/t/universal-deployer-contract-proposal/1864), 
+which deploys the contract with the given class hash and constructor arguments.
 
 The basic inputs needed for this command are:
-- Path to the compiled contract file
+- Class hash of the declared contract class, that you want to deploy
 - Network you want to target (i.e. its name or gateway URL)
-[Read the CLI reference for the deploy command](../../cli-reference.md#deploy) to learn more about all supported arguments.
 
-:::caution
-The deploy transaction will be deprecated in future StarkNet versions. 
-To deploy new contract instances, you can use the deploy syscall. 
-For more information, read [StarkNet's Contract Classes documentation](https://docs.starknet.io/docs/Contracts/contract-classes).
-:::
+[Read the CLI reference for the deploy command](../../cli-reference.md#deploy) to learn more about all supported arguments.
 
-:::info
-This command will be reworked to be compatible with the [UDC concept](https://community.starknet.io/t/universal-deployer-contract-proposal/1864). 
+:::note
```

Comment:
Maybe it should be a warning

Reply:
i wanted somewhat medium severity, because warning is red and i wanted yellow :<

Reply:
Ohhh it's _caution_